hasura.planx.uk/metadata/tables.yaml

```diff
@@ -530,6 +578,39 @@
           - data_merged
         filter: {}
         allow_aggregations: true
+    - role: demoUser
+      permission:
+        columns:
+          - created_at
+          - creator_id
+          - data
+          - id
+          - name
+          - settings
+          - slug
+          - status
+          - team_id
+          - updated_at
+          - version
+        computed_fields:
+          - data_merged
+        filter:
+          _or:
+            - _and:
+                - creator_id:
+                    _eq: x-hasura-user-id
+                - team:
+                    id:
+                      _eq: 32
+            - team:
+                _or:
+                  - id:
+                      _eq: 1
+                  - id:
+                      _eq: 29
+                  - id:
+                      _eq: 30
```

Comment:
Not sure why I didn't spot this earlier, but maybe the `_in` operator would be a good alternative here?